### DISCUSSION TITLE
Table of content without asking if links should be updated?

### DISCUSSION THREAD

**Discussion by ericrange** [dolanmiu/docx#2399] [General]
does this example: https://github.com/dolanmiu/docx/blob/master/demo/28-table-of-contents.ts

also work without the prompt if i want to update the ref links?

![image](https://github.com/dolanmiu/docx/assets/1785416/fecdd273-323e-407f-90e2-8d34c5f088f0)

**Comment by dolanmiu** [comment]
Yes this is a known issue:

https://github.com/dolanmiu/docx/issues/1212
